src/component/parser/json/converter/ProcessConverter.ts

```diff
@@ -70,9 +70,9 @@ const computeSubProcessKind = (processedSemanticType: BpmnSemanticType, bpmnElem
   }
 };
 
-const orderedFlowNodeBpmnTypes: BpmnSemanticType[] = ['adHocSubProcess', 'transaction'] // specific management for adhoc and transaction sub-processes which are handled with a dedicated ShapeBpmnSubProcessKind
+const orderedFlowNodeBpmnTypes: BpmnSemanticType[] = (['adHocSubProcess', 'transaction'] as BpmnSemanticType[]) // specific management for adhoc and transaction sub-processes which are handled with a dedicated ShapeBpmnSubProcessKind
```

Comment:
**question:** Same question as in EventDefinitionConverter.ts about  switching the syntax to "as".
We already have this discussion when I create the PR about this code, and we concluded that this was extra typing.

Reply:
Same answer, TypeScript send an error due to the removing of `[key:string]:any` 🙂